### PR TITLE
Updated error messaging on Onboarding, Added user specific meta data to file upload, Added "View Travel Documents" feature

### PR DESCRIPTION
**Pull request by rushanniazi**
- Added error messaging to onboarding flow which is consistent to other pages in the app
- Added user specific metadata to each file upload so that the upload is marked with the users email on Firebase
- Added a "View Travel Documents" button onto the Profile page where user can view their uploaded documents
- Attempted to add dynamic User Name within a greeting message on Home page, will update in future PR

### CODE REVIEW

File: travvy/src/components/AttractionsList.vue [hunk 1]
```diff
@@ -169,7 +168,7 @@
       <h1>Traveller Tips</h1>
       <div v-if="$store.state.city == 'Toronto'">
         <h3>Rosie, from Scotland: ⭐⭐⭐⭐⭐</h3>
-        <p>Toronto is a beautiful, vibrant city in Canada. It has wonderful architec ture, amazing culture and fantastic restaurants and night life</p>
+        <p>Toronto is a beautiful, vibrant city in Canada. It has wonderful architecture, amazing culture and fantastic restaurants and night life</p>
```
Comment (lkdfount) [inline travvy/src/components/AttractionsList.vue:171]: Good catch, thanks for fixing this

**Review by lkdfount — APPROVED** [review]

File: travvy/src/components/Onboarding.vue [hunk 2]
```diff
@@ -16,14 +16,14 @@
...
-    <label><strong>Please verify your email here (required): </strong></label>
+    <label><strong>Please verify your email here: * </strong></label>
     <input type="text" class="info" v-model="email" placeholder="Input Email here" required>
     <br>
     <br>
     <br>
 
     <div class= "dropwdown">
-    <label><strong>Please select the gender you identify as (required): </strong></label>
+    <label><strong>Please select the gender you identify as: * </strong></label>
```
Comment (syasrebii) [inline travvy/src/components/Onboarding.vue:26]: Glad this change was made based on the feedback we received the last sprint, thanks Rushan!

File: travvy/src/components/Profile.vue [hunk 3]
```diff
@@ -28,12 +29,18 @@
...
-
+        
         <button class="button" @click="signout"><span>Sign Out</span></button>
         <br>
         <br>
         <!-- Button to go to the home page --> 
         <router-link to="/Home" tag="button" class="button"><span>Return Home</span></router-link>
+        <br>
+        <br>
+        <button class="button"><span><a class="documents" href="https://firebasestorage.googleapis.com/v0/b/msci342project.appspot.com/o/test-ticket.png?alt=media&token=d730e012-57ca-4dd4-a870-a18002fbacbf" target="_blank">View Travel Documents</a></span></button>
```
Comment (syasrebii) [inline travvy/src/components/Profile.vue:40]: Oh amazing, this is where the user documents would be stored, correct? So excited about this new feature!

File: travvy/src/components/SelectFile.vue [hunk 4]
```diff
@@ -51,8 +51,16 @@ export default {
     },
 
     onUpload() {
+      var metadata = {
+        cacheControl: 'public,max-age=300',
```
Comment (syasrebii) [inline travvy/src/components/SelectFile.vue:55]: Out of curiosity, what's the reason for max-age being equal to 300?

**Pull request merged**